apps/demos/testing/widgets/chat/Customization.test.js

```diff
@@ -0,0 +1,130 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import { Selector } from 'testcafe';
+import { runManualTest } from '../../../utils/visual-tests/matrix-test-helper';
+import { testScreenshot } from '../../../utils/visual-tests/helpers/theme-utils';
+
+fixture('Chat.Customization')
+  .page('http://localhost:8080/apps/demos/Demos/Chat/Customization/jQuery/')
+  .before(async (ctx) => {
+    ctx.initialWindowSize = [900, 600];
+  });
+
+// runManualTest('Chat', 'Customization', ['jQuery'/* , 'React', 'Vue', 'Angular' */], (test) => {
+  test('Sending Message', async (t) => {
+    const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
```

Comment:
Can we have one test with the following chain of scenarios? 

hide avatar -> screenshot -> hide username -> screenshot -> change day header format -> screenshot -> hide day header format -> screenshot -> change timestamp format -> screenshot -> hide timestamp -> screenshot -> disable chat -> screenshot

Reply:
done